Lib/test/test_email/test_contentmanager.py

```diff
@@ -453,6 +453,45 @@ def test_set_text_11_lines_long_line_maximal_non_ascii_heuristics(self):
         self.assertEqual(m.get_payload(decode=True).decode('utf-8'), content)
         self.assertEqual(m.get_content(), content)
 
+    def test_set_text_non_ascii_with_policy_max_line_length_none_or_0(self):
+        for max_line_length in (0, None):
+            with self.subTest(max_line_length=max_line_length):
+                policy = self.policy.clone(max_line_length=max_line_length)
+                m = self.message(policy=policy)
+                content = ("áàäéèęöőáàäéèęöőáàäéèęöőáàäéèęöő"
+                        "áàäéèęöőáàäéèęöőáàäéèęöőáàäéèęöő"
+                        "áàäéèęöőáàäéèęöőáàäéèęöőáàäéèęöő.\n")
+                raw_data_manager.set_content(m,content)
```

Comment:
nit: can you please add a space after comma `(m, content)`?

Reply:
```suggestion
                raw_data_manager.set_content(m, content)
```